packages/strapi-utils/lib/build-query.js

```diff
@@ -111,16 +111,16 @@ const hasDeepFilters = ({ where = [], sort = [] }, { minDepth = 1 } = {}) => {
 
 const normalizeWhereClauses = (whereClauses, { model }) => {
   return whereClauses
-    .filter(({ value }) => !_.isNull(value))
-    .map(({ field, operator, value }) => {
-      if (_.isUndefined(value)) {
-        const err = new Error(
-          `The value of field: '${field}', in your where filter, is undefined.`
-        );
-        err.status = 400;
-        throw err;
+    .filter(({ field, value }) => {
+      if (_.isNull(value)) {
```

Comment:
I was thinking we should allow null values actually 🤔

Reply:
It is filtered out because of the `id_null`, this is when a developer specifically wants to look for `null` and not accedentily have the value `null` because of wrong logic/bug.